templates/zerver/integrations/onyx.md

```diff
@@ -0,0 +1,34 @@
+# Zulip Onyx Integration
+
+Export discussions from Zulip channels and topics to your Onyx knowledge
+base!
+
+{start_tabs}
+
+1.  {!create-a-generic-bot.md!}
+
+1. [Subscribe the bot][subscribe-channels] to the Zulip channels that you
+   want to export.
+
+1. In Onyx, open the **Admin Dashboard** and select the **Zulip Connector**.
+
+1. Copy [your Zulip bot's credentials]({{ zulip_url }}/#settings/your-bots)
```

Comment:
Rewrote this instruction by merging the previous "Copy" instruction which was placed below the "create a generic bot" instruction. With the addition of the link to the "Settings > Your Bots" page, this is feasible.
Both the versions are available as screenshots in the PR description.

To clarify that context switch between Zulip and Onyx in the same instruction, I've mentioned "in Zulip", "in Onyx". Does that suffice? Or should we switch back to using separate instructions.

Reply:
I don't have a strong opinion. The split version might be fine, but you should swap steps (2) and (3) there.